deployment/docker-compose/docker-compose.dev.yml

```diff
@@ -9,12 +9,11 @@ services:
     image: pgvector/pgvector:pg16
     restart: always
     env_file:
-      - .env
+      - .core_backend.env
```

Comment:
I had thought about having another `template.db.env` for DB credentials, but it felt bloated. Hence, put the DB credentials in `.core_backend.env` and passing them here!

Reply:
That makes sense since only the backend communicates to the DB for now.